libjob_queue/src/lsf_driver.c

```diff
@@ -228,6 +228,51 @@ int lsf_job_parse_bsub_stdout(const char * bsub_cmd, const char * stdout_file) {
   return jobid;
 }
 
+/**
+ * Assumes fname points to a file with content "hname1:hname2:hname3" as written by lsf_job_write_bjobs_to_file
+ */
+stringlist_type * lsf_job_parse_bjobs_file(const char* fname) {
+  FILE *stream = util_fopen(fname, "r");
+  stringlist_type * hosts = NULL;
+  bool at_eof = false;
+  while (!at_eof) {
+    char * line = util_fscanf_alloc_line(stream, &at_eof);
+    if (line != NULL) {
+      hosts = stringlist_alloc_from_split(line, ":"); // bjobs uses : as std. delimiter
```

Comment:
The sanitization of the `bjobs` output must also remove the trailing `n*` in front of the hostnames; i.e. for a parallell running with four CPU's on two hosts the raw output from `bjobs` can be:
```txt
4*host1:4*host2
```
we want to extract that to `{"host1", "host2"}`. Not sure if a "1*" is present for a scalar job.